server/core/templates/base_htmx_with_nav.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
```

Comment:
Minor but this can just extend the base_htmx as well. You can nest templates :)

Reply:
Resolved by [00df717](https://github.com/kanidm/kanidm/pull/2868/commits/00df7171ef215c16b1a66e5d7c1a514a1f9b3d95)